android/app/src/main/java/com/sample/ComponentActivity.kt

```diff
@@ -0,0 +1,29 @@
+package com.sample
```

Comment:
Do we have to use a proper namespace here?

Reply:
Hm, what do you mean? The package name corresponds to the file system path, which is `com/sample` in this case :P

Reply:
I guess what I'm trying to ask is whether we should consider having a proper namespace. I know that it reflects the file structure.

Reply:
Aha, I see. I prefer to namespace classes by "features" and when there is more than one class to put into a package. `ComponentActivity` does not have any logically correlated classes at the moment, but there will be a few in the future (when support for the native controllers lands). I would prefer to do it then, unless you have some other ideas?

Reply:
https://github.com/microsoft/react-native-test-app/issues/34